src/pages/RepoPage/CoverageOnboarding/UseFrameworkInstructions.tsx

```diff
@@ -0,0 +1,174 @@
+import { useMemo } from 'react'
+
+export type Framework = 'Jest' | 'Vitest' | 'Pytest' | 'Go'
+export type FrameworkInstructions = ReturnType<typeof UseFrameworkInstructions>
+
+interface UseFrameworkInstructionsArgs {
+  orgUploadToken?: string | null
+  owner: string
+  repo: string
+}
+
+export function UseFrameworkInstructions({
+  orgUploadToken,
+  owner,
+  repo,
+}: UseFrameworkInstructionsArgs) {
+  return useMemo(
+    () => ({
+      Jest: {
+        install: 'npm install --save-dev jest',
+        run: 'npx jest --coverage',
+        githubActionsWorkflow: `name: Run tests and upload coverage
```

Comment:
These seem to have lost the indentation it formerly had - does that make anything ugly?

Reply:
omg totally missed this. Very nice catch 🥔